### DISCUSSION TITLE
Rad Pro: Haptic pulse on startup

### DISCUSSION THREAD

**Discussion by ThatBum** [Gissio/radpro#166] [Ideas]
Most devices with haptic feedback do this, presumably to prove the haptic motor is working, or to confirm that the device has entered an early boot stage. The stock firmware does this. Hell, most smartphones do this. Would be a fairly trivial addition. Especially useful for this device since haptic alarm may be safety critical.

**Comment by ugopapetti** [comment]
Good point, agree.

ugo

Il sab 7 dic 2024, 04:38 ThatBum ***@***.***> ha scritto:

**Comment by ugopapetti** [comment]
About the power-on, a possibile improvement could be execute a POST (Power
on self test) including led test, clicker test, alarm sound, battery
voltage (even It available on every screen), hv&detector test(i.e. First
click detector) and other I did not mention (Memory size, available space
for data recording, ...). In case too long, could be bypassed by clicking
another time the Power on button

This Will add valute to  the system.
ugo

Il sab 7 dic 2024, 07:41 Ugo Papetti ***@***.***> ha scritto:

**Comment by ThatBum** [comment]
A beep at startup would be good as well, the stock firmware also does this. LED flash too. Free space/time for data logging would make a good addition to the statistics page but sounds more involved to implement and a little out of scope of this simple request. There's already continuous fault detection for the tube/HV so it's not really necessary to run this at startup.

The point of activating these things at startup is that the system can't really know if they are working or not (other then measuring the back-EMF of the motor or something esoteric like that) so it relies on the human operator to feel, listen, and look for them respectively. The splash screen itself could be considered a display test.

**Comment by Gissio** [comment]
A short vibrator and LED test on power up is a great idea and easy to implement.

Sound, however, is not. I can think of many scenarios where someone might not be interested in drawing attention.

**Reply by ThatBum** [reply]
Yeah this did belatedly occur to me. Don't want to alarm flea market people for instance. Thanks for the quick replies btw!
